package.json

```diff
@@ -67,7 +67,7 @@
     "react-dom": "^16.8.6 || ^17.0.0 || ^18.0.0"
   },
   "dependencies": {
-    "@sendbird/chat": "^4.3.0",
+    "@sendbird/chat": "^4.8.3",
```

Comment:
nit: can we set the required version to `4.8.4`? because there was a hotfix

Reply:
Yeah updated it in here https://github.com/sendbird/sendbird-uikit-react/pull/551/commits/036428ded890f8eff5cb7ebc0cc6a3b3cdec320d